enhancement/store_artifacts

losete opened the pull request:
Pull Request made for storing the artifacts generated during the build and all the reports generated

losete committed:
Added 'post' section to store the artifacts

losete committed:
Added 'Prepare artifacts' stage

losete committed:
Deleted 'Prepare artifacts' stage + Added deploy to artifacts

losete committed:
Renamed 'deploy' to 'build'

sebascm commented on Jenkinsfile:
```diff
@@ -31,8 +31,15 @@ pipeline {
\ No newline at end of file
+  post {
+    always {
+      archiveArtifacts 'reports.tar.gz'
```
I suggest using onlyIfSuccessful: true 
With that, if CI fails, it won't make the artifact.

losete replied:
I think that with reports is interesting to have a report, it may tell you what have failed, but it is an interesting idea to apply it to build

sebascm replied:
Yes, it's better using it only with the build

sebascm reviewed (changes requested):
I suggest using onlyIfSuccessful: true
With that, if CI fails, it won't make the artifact.

losete committed:
Added parameter onlyIfSuccesful

This is done because we do not want the artifact to be uploaded if the CI fails
